wasm/src/lib.rs

```diff
@@ -56,28 +57,13 @@ fn get_qir_internal(code: &str) -> std::result::Result<String, (Error, Vec<Frame
     // Ensure it compiles before trying to add it to the store.
     if !errors.is_empty() {
         // This should never happen, as the program should be checked for errors before trying to
-        // generate code for it. But just in case, simply report the entire source as 'fail to generate output'
-        return Err((
-            Error::OutputFail(Span {
```

Comment:
This line was added _after_ I made my changes and was causing a build break because I now require `PackageSpan` 😭 . Given that the error is immediately thrown out and converted to a string in the caller, I changed it to be a hardcoded string...

Reply:
I mentioned this on Bill's PR, but I think we want to refactor some here to make sure that there is common behavior between this code and `run_internal` in terms of how compile errors from the provided code string are handled. There is an existing backlog issue for it in the commont on `run_internal` ([#149](https://github.com/microsoft/qsharp/issues/149))